jxl/src/frame/group.rs

```diff
@@ -7,17 +7,220 @@ use crate::{
     bit_reader::BitReader,
     error::{Error, Result},
     frame::{
-        block_context_map::*, coeff_order, transform_map::*, HfGlobalState, HfMetadata,
-        LfGlobalState,
+        block_context_map::*, coeff_order, quant_weights::DequantMatrices, transform_map::*,
+        HfGlobalState, HfMetadata, LfGlobalState,
     },
     headers::frame_header::FrameHeader,
     image::{Image, Rect},
-    util::tracing_wrappers::*,
-    util::CeilLog2,
-    var_dct::transform::*,
+    util::{tracing_wrappers::*, CeilLog2},
+    var_dct::{
+        dct::compute_scaled_dct,
+        dct_scales::{dct_total_resample_scale, DctResampleScales, HasDctResampleScales},
+        transform::*,
+    },
     BLOCK_DIM, BLOCK_SIZE,
 };
 
+// Computes the lowest-frequency LF_ROWSxLF_COLS-sized square in output, which
```

Comment:
where is LF_ROWS / LF_COLS defined? isn't that just ROWS / COLS below?

Reply:
Oh, yes, they were additional parameters in the C++ which we `static_assert`ed to be equal to `ROWS` / `COLS` so I figured we might as well just not take them. Thanks.